tools/control_board/generate_control_board.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python2
```

Comment:
PEP0394 states to only use `python` if it is compatible with both python3 and python2.
This script is not.
If I change it to python, my system where python is a link to python3, will not work.

I am using pep394 correctly here by using python2. I don't know why you're pushing for this change when it breaks more than it fixes.